moveit_core/utils/include/moveit/utils/moveit_error_code.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <moveit_msgs/MoveItErrorCodes.h>
+
+namespace moveit
+{
+namespace core
+{
+/**
+ * @brief a wrapper around moveit_msgs::MoveItErrorCodes to make it easier to return an error code message from a function
+ */
+class MoveItErrorCode : public moveit_msgs::MoveItErrorCodes
+{
+public:
+  MoveItErrorCode()
+  {
+    val = 0;
+  }
+  MoveItErrorCode(int code)
+  {
+    val = code;
+  }
+  MoveItErrorCode(const moveit_msgs::MoveItErrorCodes& code)
+  {
+    val = code.val;
+  }
```

Comment:
Would it also be useful to add a `operator moveit_msgs::MoveItErrorCodes()` to this?

Reply:
I don't think this is necessary because the class inherits from it already.